modules/nft/src/lib.rs

```diff
@@ -556,3 +550,25 @@ impl<T: Config> NFT<T::AccountId> for Pallet<T> {
 		Self::do_transfer(from, to, token)
 	}
 }
+
+impl<T: Config> ManageNFT<T::AccountId, CID, Attributes> for Pallet<T> {
+	type ClassId = ClassIdOf<T>;
+	type TokenId = TokenIdOf<T>;
+
+	/// To mint new NFT tokens.
+	fn mint(
```

Comment:
I think we can have `mint` and `mint_batch` to make interface easier to use.
We only have one for extrinsic because there is nothing to return.